src/App.jsx

```diff
@@ -1,15 +1,21 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import { Route, Routes, BrowserRouter } from 'react-router-dom';
+import Cities from './routes/cities.route';
+import Countries from './routes/countries.route';
+import CityDetail from './routes/city-detail';
+import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <div className="App">
-      
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Countries />} />
+          <Route path="cities" element={<Cities />} />
+          <Route path="city-detail" element={<CityDetail />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
- You have done well so far, but your app is missing some little design features as seen in the design template. Please note your nav bar is missing the settings and mic icon. Please fix this 👍 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Design template:
![xdjxj](https://user-images.githubusercontent.com/73607512/225966084-cc4ccba5-67ca-42ab-90ba-f6c335c26263.PNG)

Your project:
![nset](https://user-images.githubusercontent.com/73607512/225966337-72d2c4d5-6378-4e5d-a60c-47665fb75bb6.PNG)

Reply:
Well resolved 🚀 . Great job !!! 🍾 

![image](https://user-images.githubusercontent.com/38088818/226092052-bdc31dc8-7269-44b2-a689-abd4460aa179.png)